x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_overview/utils/format_helpers.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const formatNumber = (val: number) => {
```

Comment:
Have you checked `formatNumber` from  `@elastic/eui`?

Reply:
Good call @yngrdyn  - switched over and the `decimal0` format is even better than what I have now:

<img width="180" alt="Screenshot 2025-03-20 at 17 54 14" src="https://github.com/user-attachments/assets/a1d20ce4-a789-4a7e-ab86-027dc76df198" />